scripts/trivy_scan.sh

```diff
@@ -35,8 +35,9 @@ fi
 echo "Printing trivy ignore file" 
 cd "$GITHUB_WORKSPACE" && cat .trivyignore
 
+
 echo "Checking for low level vulnerablities" 
-eval "trivy --exit-code 0 --severity=HIGH,MEDIUM,LOW,UNKNOWN $IMAGE"
+eval "trivy image --exit-code 0 --severity=HIGH,MEDIUM,LOW,UNKNOWN $IMAGE"
```

Comment:
Can you remove the severity to show everything, since the next call will hide the unfixed ones?